tests/components/html5/test_notify.py

```diff
@@ -87,7 +87,7 @@ def test_get_service_with_no_json(self):
 
         assert service is not None
 
-    @patch("pywebpush.WebPusher")
+    @patch("pywebpush.pywebpush.WebPusher")
```

Comment:
No, we need to use the path in the homeassistant package where we use this class.

```py
"homeassistant.components.html5.notify.WebPusher"
```

Reply:
Ok will do.

Sorry I have big problems to understand tests patching and the circular dependancy in Python. I'm coming from the Java side of the devs :roll_eyes: :grimacing: